mailboxes/lang/de

```diff
@@ -233,73 +234,73 @@ send_efrom=Fehlende Absenderadresse (FROM)
 send_esubject=Fehlender Betreff
 send_title=Mail versandt
 send_title2=Mail gesichert
-send_ok=Mail erfolgreich versandt an $1
-send_sending=Sende Mail an $1 ..
-send_ecannot=Sie dürfen im Namen dieses Benutzers keine Mail versenden
+send_ok=Mail erfolgreich an $1 versendet
+send_sending=Sende Mail an $1
+send_ecannot=Sie dürfen Mails im Namen dieses Benutzers nicht versenden
 send_esmtp=SMTP-Kommando $1 gescheitert : $2
-send_eattach=Die Dateianhänge dürfen zusammen nicht mehr als $1 KiB groß sein.
+send_eattach=Die Dateianhänge dürfen zusammen nicht mehr als $1 KB groß sein
 send_eperms=Benutzer $1 kann $2 nicht lesen
-send_eperms2=Sie dürfen die Datei $1 nicht versenden
-send_epath=Das Sendmail-Binary $1 existiert nicht.
+send_eperms2=Die Datei $1 darf nicht versendet werden
+send_epath=Das Sendmail-Binary $1 existiert nicht
 send_efile=Konnte Dateianhänge nicht lesen $1 : $2
-send_done=.. fertig.
+send_done=Fertig.
```

Comment:
The standard way to show progressive output in Webmin is:

```
Running the example command ..
.. done
```

or

```
Running the example command ..
.. error : exited with code 128
```

So this change should be reverted.